media-proxy/include/session-rdma.h

```diff
@@ -0,0 +1,86 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 Intel Corporation
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#ifndef __SESSION_RDMA_H
+#define __SESSION_RDMA_H
+
+#include <stdlib.h>
+#include <mcm_dp.h>
+
+#include "libfabric_dev.h"
+#include "libfabric_ep.h"
+#include "session-base.h"
+using namespace std;
```

Comment:
Avoid using `std` this way. As @Sakoram suggested, switch to passing it explicitly, e.g. `std::mutex`.

Reply:
done